### DISCUSSION TITLE
Started on a veroboard version.

### DISCUSSION THREAD

**Discussion by ixmeins** [picocomputer/community#7] [Q&A]
![bbpico6502](https://github.com/picocomputer/discussions/assets/13176915/d183e5cc-e202-477e-8397-4ac537b30bf7)

**Comment by ixmeins** [comment]
So it partially lives. I had a fake 65c22 and 65c02 from a fleabay cretin (bought in 2020) so I've just ordered a couple of real ones from Mouser, should be here in a few days. I used 0603 resistors to make the VGA  D/A networks. Nice clean image. 
![20230604_234519](https://github.com/picocomputer/discussions/assets/13176915/03e673cf-457d-409b-9279-81c646ca0e64)
![20230604_234657](https://github.com/picocomputer/discussions/assets/13176915/3683d041-fd64-467e-b44e-f1e213878351)
![20230604_234625](https://github.com/picocomputer/discussions/assets/13176915/cfab11b6-9c9d-4051-a46b-cedfa76beed4)

**Comment by rumbledethumps** [comment]
Your UMC "65C02" has a WDC part number. Definitely suspect.

Those splintered board edges take me back. FR4 is for wimps.

**Reply by ixmeins** [reply]
I've been using Vero since the 70's and yet have not built anything on veroboard for probably a decade now. You get very used to fast commercial PCB or milling them on a LPKF etc. Back to basics can be good! Can't wait for the new processor etc to arrive.

**Comment by ixmeins** [comment]
Borrowed a real 65c02 from a Canberra (Australia) local and the beast fired up straight away. Might get to try installing vscode stuff later today (after noon now).  Winner winner chicken dinner as mr squeak says (eevblog).

**Comment by rumbledethumps** [comment]
Be sure to swap PHI2 and PWMR as shown on the latest schematic. Don't panic, there won't be any more changes since I have working circuit boards with this new pinout.

The only other change of note is pin 9 of the VGA connector now has power. Turns out VGA-HDMI adapters will use if it's there. If not, the adapter will need external power.

**Reply by ixmeins** [reply]
You mean RESB and PWML...
